website/aws.erb

```diff
@@ -1876,6 +1876,16 @@
                     </ul>
                 </li>
 
+                <li>
+                    <a href="#">MSK Resources</a>
```

Comment:
Nit: I'm thinking we might want to call out the full name here for folks searching for just "kafka", e.g.

```suggestion
                    <a href="#">Managed Streaming for Kafka (MSK) Resources</a>
```